### PR TITLE
Update introduction.md

### PR DESCRIPTION
**Pull request by freducom**
added material design

[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/132)

### CODE REVIEW

File: src/main/markdown/doc/latest/polymer-tutorial/introduction.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## **Disclaimer** <a id="pol-disclaimer"></a>:
 _This tutorial is based on the usage of three third-party **open-source** libraries, external to, and not managed by, the **gwtproject** organization_:
 
-* _[**Polymer**][1]: a JavaScript library for building web applications with Web Components. Polymer is owned by **Google Inc**._
+* _[**Polymer**][1]: a JavaScript library for building web applications with Web Components. Poylmer is the implementation of Google's Material Design. Polymer is owned by **Google Inc**._
```
Comment (jDramaix) [inline src/main/markdown/doc/latest/polymer-tutorial/introduction.md]: typo: Polymer
And  the sentence is wrong. Polymer is not the implementation of Google's Material Design. Paper elements are the polymer implementation of Material design.

Comment (freducom) [inline src/main/markdown/doc/latest/polymer-tutorial/introduction.md]: Good point! Also: Could you add there "Polymer Paper is the implementation of Google's Material Design". 

=> The point being that people looking for material design for GWT can find this page.

**Comment by jDramaix** [comment]
@freducom Could you sign the CLA and make the change ?

**Comment by jDramaix** [comment]
ping

**Comment by jnehlmeier** [comment]
Closing because of missing CLA and no activity for 2+ months.

**Pull request closed without merging**